### PR TITLE
[TGBOT-152] Move the UUID to the repository

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-152

### CODE REVIEW

**Review by AlexanderYarokhovich — DISMISSED** [review]
I would just call IdGenerator

**Comment by schneidervs** [comment]
Can resolve conflicts immediately

**Review by schneidervs — APPROVED** [review]

**Review by yurasava — DISMISSED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]

**Comment by schneidervs** [comment]
The isContains method does not handle a null value in some repositories

File: stoservice/src/test/java/com/tutrit/stoservice/repository/UserRepositoryTest.java [hunk 1]
```diff
@@ -58,9 +57,9 @@ void deleteUserById() {
 
     @Test
     void deleteUser() {
-        int expectedCount = userRepository.count();
+        int expectedCount = userRepository.count() + 1;
```
Comment (yurasava) [inline stoservice/src/test/java/com/tutrit/stoservice/repository/UserRepositoryTest.java:60]: Why did you add " + 1 " ?

File: stoservice/src/main/java/com/tutrit/stoservice/repository/CarRepository.java [hunk 2]
```diff
@@ -84,6 +89,12 @@ public boolean isContains(String id) {
         return car != null;
     }
 
+    @Override
+    public void setUUID(Car car) {
+        car.setId(UUID.randomUUID().toString());
```
Comment (AlexanderYarokhovich) [inline stoservice/src/main/java/com/tutrit/stoservice/repository/CarRepository.java:94]: Why translate UUID to String&

**Review by AlexanderYarokhovich — DISMISSED** [review]

**Review by yurasava — APPROVED** [review]

**Review by schneidervs — APPROVED** [review]

**Review by yurasava — APPROVED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]
